### ISSUE TITLE
SC2169: "In dash, ulimit [...] is not supported" (but it is)

### ISSUE THREAD

**Issue by j1elo** [koalaman/shellcheck#1406]
#### For bugs
- Rule Id: `SC2169`
- My shellcheck version (`shellcheck --version` or "online"): online
- [X] I read the issue's wiki page, https://github.com/koalaman/shellcheck/wiki/SC2169
- [X] I tried on shellcheck.net and verified that this is still a problem

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/dash
ulimit -c unlimited
ulimit -n 100000
ulimit -Sp 3600

```

#### Here's what shellcheck currently says:

```
SC2169: In dash, ulimit -c is not supported.
SC2169: In dash, ulimit -n is not supported.
SC2169: In dash, ulimit -S is not supported.
```

#### Here's what I wanted or expected to see:

No warning; in dash, `ulimit` commands **are** supported:

```
$ man dash
(search "ulimit")
ulimit [-H | -S] [-a | -tfdscmlpn [value]]
            Inquire about or set the hard or soft limits on processes or set new limits.  The choice between hard limit (which no process is allowed to violate, and which may not be raised once it has been
            lowered) and soft limit (which causes processes to be signaled but not necessarily killed, and which may be raised) is made with these flags:

            -H          set or inquire about hard limits

            -S          set or inquire about soft limits.  If neither -H nor -S is specified, the soft limit is displayed or both limits are set.  If both are specified, the last one wins.

            The limit to be interrogated or set, then, is chosen by specifying any one of these flags:

            -a          show all the current limits

            -t          show or set the limit on CPU time (in seconds)

            -f          show or set the limit on the largest file that can be created (in 512-byte blocks)

            -d          show or set the limit on the data segment size of a process (in kilobytes)

            -s          show or set the limit on the stack size of a process (in kilobytes)

            -c          show or set the limit on the largest core dump size that can be produced (in 512-byte blocks)

            -m          show or set the limit on the total physical memory that can be in use by a process (in kilobytes)

            -l          show or set the limit on how much memory a process can lock with mlock(2) (in kilobytes)

            -p          show or set the limit on the number of processes this user can have at one time

            -n          show or set the limit on the number files a process can have open at once

            -r          show or set the limit on the real-time scheduling priority of a process
```

Also, my script works without errors, so `dash` is definitely accepting these commands.

System info:
```
$ lsb_release -d
Description:	Linux Mint 18.3 Sylvia
```
(This is the same as Ubuntu 16.04 Xenial)
